entrypoint.sh

```diff
@@ -30,8 +30,13 @@ run_test_if_needed() {
     echo 'Skip test'
   else
     echo "Run test"
-    flutter pub get
-    flutter test
+    if "${INPUT_FLUTTER_PACKAGE}"; then
```

Comment:
Thanks for fixing!
Only one point.
I want a method like "Publish Package".

Because you can tell what you are doing by the method name 😄